### DISCUSSION TITLE
Idea to fix 'option clash for package mhchem' error

### DISCUSSION THREAD

**Discussion by britthubs** [GQCG-edu/master-thesis-article#8] [Ideas]
Hello
I have found that when trying to compile in VSCode there seems to be a clash with the mhchem and chemmacros packages, which results in the latex code not compiling (It gives the following error: Option clash for package mhchem.) I do want to note that in Overleaf the code does compile, but the same error is given (this is only after creating the copy and not changing anything, it doesn't seem to appear in the read-only state).

After looking for a possible solution I found that mhchem is already included in the chemmacros package and can be added in this way ([source](https://ctan.math.washington.edu/tex-archive/macros/latex/contrib/chemmacros/chemmacros-manual.pdf)):
```
\usepackage{chemmacros}
\chemsetup{formula = mhchem}
``` 
Instead of:
```
\usepackage{chemmacros}
\usepackage{mhchem}
``` 
This has removed the error for me.
*Disclaimer: I am not entirely sure if this would overwrite something else in the chemmacros package, I have tested if ```\ce{}``` works and that seems to be the case, but I haven't tested anything else.*

**Comment by ruvdrsti** [comment]
Hi

Thank you for noticing this error. However, when I compile this in the provided devcontainer it works as intended. Could you verify this? If you are not familiar with devcontainers, you can read more on [this page](https://github.com/GQCG-edu/bootcamp/blob/master/training/local.md).  If you need additional help with this, you are always welcome to come by our office (S3 third floor).

We look forward to hearing from you.

**Reply by britthubs** [reply]
Compiling it in the devcontainer indeed fixed the error, thank you! The error does still appear in overleaf after making a clean copy as described in the instructions. This is the full message:
<img width="614" alt="image" src="https://github.com/user-attachments/assets/142ba57e-62b5-41af-b175-62dac5e9e6ed">
Thank you again for taking the time and linking the extra information, it was helpful!

**Reply by ruvdrsti** [reply]
Happy to help! Does your fix also work in the container? If so, feel free to open a pull request on this repo with either myself or @emielvb as a reviewer.
